.github/workflows/node.js-tests-upcoming.yml

```diff
@@ -68,6 +68,7 @@ jobs:
       fail-fast: false
       matrix:
         node-version: [18.x]
+        mongodb-version: [6.0]
```

Comment:
Does this have to be in the matrix? I do not see us testing in multiple versions of MongoDB, and it means we have to update every open PR for the updated check status this entails.

Reply:
An alternative would be to use an exact match in the task stanza as an environment variable. I do not know your reasoning here, so I'll defer to what you think is better without using the matrix.

Reply:
Nope, I just assumed you'd want to! I'll take them out.